### PR TITLE
Best practices on how to do structured logging

### PR DESCRIPTION
**Pull request by alenkacz**

### CODE REVIEW

**Review by meln1k — APPROVED** [review]

File: docs/code-culture.md [hunk 1]
```diff
@@ -136,3 +137,30 @@ For example:
...
 * A validation function should return validation errors, not throw them.
 * A storage module will throw an exception if a connection is unavailable.
+
+## Structured logging
+
+Debugging distributed system is hard. To make this at least a bit easier we favor log messages including logging context (using structured logging). Core USI is providing all necessary tooling for framework developer to log in structured way.
+
+For introduction to structured logging read [this article](https://stackify.com/what-is-structured-logging-and-why-developers-need-it/).
+
+What should always be part of the logging message:
```
Comment (zen-dog) [inline docs/code-culture.md]:
```suggestion
All relevant ids should be part of the logging message e.g.:
```

File: docs/code-culture.md [hunk 2]
```diff
@@ -136,3 +137,30 @@ For example:
...
+
+Example of good framework log messages (a little bit simplified) can look like example below:
+
+```json
+{
+  "time": "1.1.1970",
+  "level": "INFO",
+  "podId": "my-pod",
+  "taskId": "mesos-task-my-pod-1",
+  "message": "Pod 'my-pod' was killed"
```
Comment (zen-dog) [inline docs/code-culture.md]: I would absolutely include `taskId` into the message. 
```suggestion
  "message": "Task $taskId of Pod $podId was killed"
```

**Comment by zen-dog** [comment]
Lgtm. A few nits though

**Review by kensipe — COMMENTED** [review]
I like Alex's suggested changes!

**Review by zen-dog — APPROVED** [review]

**Pull request merged**